vng_api_common/inspectors/query.py

```diff
@@ -3,7 +3,7 @@
 try:
     from django.utils.encoding import force_str
 except ImportError:  # Django < 4.0
-    from django.utils.encoding import force_text as force_str
+    from django.utils.encoding import force_str as force_str
```

Comment:
I think it should all be reduced to 1 line:
`from django.utils.encoding import force_str`